packages/core/rollup.config.mjs

```diff
@@ -74,6 +74,7 @@ export default ["r4b", "r5"].flatMap((release) =>
           format: format === "esm" ? "m" : "c",
         },
       ],
+      external: ['@bonfhir/fhirtypes/r4b', '@bonfhir/fhirtypes/r5'],
```

Comment:
@opiation marking these dependencies as external resolves the warning but at the same time, we _want_ to include the types from fhirtypes in the bundle

Reply:
I'm confused. My understanding of Rollup's [`external` config]() is that packages listed there are _not_ bundled with the code being _rolled up_. Sounds to me like that means that `@bonfhir/core` won't include the types from `@bonfhir/fhirtypes` that it re-exports. You mention though that it _is_ included. How did you validate that? Essentially, we _should_ be able to import something like `Patient` from the generated `@bonfhir/core` bundle like:

```ts
// in /packages/core/import_test.ts

import type { Patient } from "./dist/r4b/esm/index";
//            ^^^ Should not be `any`
```

Reply:
@opiation the key thing to note is that the `input` of the ts rollup config is `dist/${release}/${format}/dts/index.d.ts` .. the output of the js rollup config that _does_ include bonfhir/fhirtypes. I will look into adding a test suite in core to explicitly test the re-exported types

Reply:
Tested and confirmed that functions and re-exported types are loaded correctly. So we are good to merge 👍